src/views/ConfigurationVideoView.vue

```diff
@@ -217,6 +251,17 @@ const downloadAndUpdateDB = async (filenames: string[]): Promise<void> => {
   selectedFilesNames.value = []
 }
 
+async function playVideoOnModal(videoFileName: string[]): Promise<void> {
+  const videoBlob = await videoStore.videoStoringDB.getItem(videoFileName[0])
+  if (videoBlob instanceof Blob) {
+    const tempFileUrl = URL.createObjectURL(videoBlob)
+    videoFile.value = tempFileUrl
+    openVideoPlayerDialog.value = true
+  } else {
+    console.error('Video data is not a Blob:', videoBlob)
+  }
```

Comment:
```suggestion
  if (!(videoBlob instanceof Blob)) {
    console.error('Video data is not a Blob:', videoBlob)
    return
  }
  const tempFileUrl = URL.createObjectURL(videoBlob)
  videoFile.value = tempFileUrl
  openVideoPlayerDialog.value = true
```

Apply early return on failures to avoid extra indentation and unnecessary else.